client/configuration_variable.go

```diff
@@ -42,7 +42,7 @@ type ConfigurationVariable struct {
 	Type           *ConfigurationVariableType   `json:"type,omitempty"`
 	Schema         *ConfigurationVariableSchema `json:"schema,omitempty"`
 	ToDelete       *bool                        `json:"toDelete,omitempty"`
-	IsReadonly     *bool                        `json:"isReadonly,omitempty"`
+	IsReadOnly     *bool                        `json:"isReadonly,omitempty"`
```

Comment:
renamed it (in multiple places) from `IsReadonly` to `IsReadOnly` (since the terraform field is `is_read_only`).